classes/migration/install/SubmissionsMigration.php

```diff
@@ -257,6 +257,8 @@ public function up(): void
             $table->dateTime('date_due')->nullable();
             $table->bigInteger('created_by')->nullable()->default(null);
             $table->foreign('created_by')->references('user_id')->on('users');
+            $table->bigInteger('started_by')->nullable()->default(null);
+            $table->foreign('started_by')->references('user_id')->on('users')->cascadeOnDelete();
```

Comment:
Here even though we make it nullable , still defining rule to delete the row as `cascadeOnDelete` . so are we not going to preserve the task when the user who started get deleted ? I am not sure about the intended workflow but is it not possible to have a task ongoing even when the user who started it got removed from the system ?